.vscode/settings.json

```diff
@@ -0,0 +1 @@
+123,125
```

Comment:
I believe this file does not belong to the device configuration. My suggestion is to remove it from the PR.